### DISCUSSION TITLE
Visualization of a Prime sieve calculation

### DISCUSSION THREAD

**Discussion by fvbakel** [PlummersSoftwareLLC/Primes#577] [Show and tell]
Visualization of a Prime sieve calculation.

I have made a Visualization of the calculation of the sieve with the base algorithm. You can find it here: https://www.youtube.com/watch?v=DK_i16yKVMw

It would be nice if we can make some more visualizations of the more advanced algorithms.

**Comment by ItalyToast** [comment]
It might be intressting but it doesnt look very intuitive. Maybe if the sieve was smaller so the number squares got bigger or showing the numbers on a line instead of a square similar to the sorting algorithms could make it better.

**Reply by fvbakel** [reply]
@ItalyToast Thanks for your feedback. There are already multiple much better visuals for smaller sieves on YouTube, so wanted to do something different. If you look at the code that was used you can see that the actual image of each frame is used as storage.
